### DISCUSSION TITLE
Private pages are browsable even when user is not logged-in

### DISCUSSION THREAD

**Discussion by vipinc007** [requarks/wiki#4576] [Help / Questions] [answered]
Hello, I am facing a small issue with private and public pages. 
My private pages are still browsable even when the user is not logged-in. Is it possible to make only the public pages visible when a user is not logged in and not show them the private pages. My requirement is to show the private pages only when user is logged-in.  
Waiting to hear from you guys

**Comment by NGPixel** [comment, marked as answer]
Check https://docs.requarks.io/groups for how permissions work. You need to change Guests group permissions.
